packages/webapp/pages/welcome/index.tsx

```diff
@@ -0,0 +1,112 @@
+import React, { ReactElement, useEffect } from 'react';
+import Feed, { FeedProps } from '@dailydotdev/shared/src/components/Feed';
+import { ANONYMOUS_FEED_QUERY } from '@dailydotdev/shared/src/graphql/feed';
+import {
+  OtherFeedPage,
+  generateQueryKey,
+} from '@dailydotdev/shared/src/lib/query';
+import { useAuthContext } from '@dailydotdev/shared/src/contexts/AuthContext';
+import { ViewSize, useViewSize } from '@dailydotdev/shared/src/hooks';
+import { useRouter } from 'next/router';
+import { onboardingUrl } from '@dailydotdev/shared/src/lib/constants';
+import Logo, { LogoPosition } from '@dailydotdev/shared/src/components/Logo';
+import {
+  Button,
+  ButtonSize,
+  ButtonVariant,
+} from '@dailydotdev/shared/src/components/buttons/Button';
+import { AuthTriggers } from '@dailydotdev/shared/src/lib/auth';
+import { cloudinary } from '@dailydotdev/shared/src/lib/image';
+import classNames from 'classnames';
+import { NextSeo, NextSeoProps } from 'next-seo';
+import { authGradientBg } from '@dailydotdev/shared/src/components/auth';
+import { getLayout as getFooterNavBarLayout } from '../../components/layouts/FooterNavBarLayout';
+import { getLayout } from '../../components/layouts/FeedLayout';
+import { defaultOpenGraph, defaultSeo, defaultSeoTitle } from '../../next-seo';
+
+const seo: NextSeoProps = {
+  title: defaultSeoTitle,
+  openGraph: { ...defaultOpenGraph },
+  ...defaultSeo,
+};
+
+const DemoPage = (): ReactElement => {
+  const router = useRouter();
+  const { user, showLogin } = useAuthContext();
+  const isLaptop = useViewSize(ViewSize.Laptop);
+
+  const feedProps: FeedProps<void> = {
+    feedName: OtherFeedPage.Welcome,
+    feedQueryKey: generateQueryKey(OtherFeedPage.Welcome, user),
+    query: ANONYMOUS_FEED_QUERY,
+    pageSize: 10,
+    allowFetchMore: false,
+  };
+
+  useEffect(() => {
+    if (isLaptop) {
+      router.replace(onboardingUrl);
+    }
+  }, [isLaptop, router]);
+
+  return (
+    <>
+      <NextSeo {...seo} />
+      <div
+        className={classNames(
+          'sticky top-0 z-max flex h-12 w-full justify-between border-b border-accent-cabbage-default px-4 py-2',
```

Comment:
Enginnering decision, its always purple I don't know of way of making it purple only when stick (scrolls from offset), except from JS but that seems overkill here. 
<img width="355" alt="image" src="https://github.com/user-attachments/assets/1d5f7551-1192-4717-90fb-9705b36e18f7">